src/sb_handle/portblas_handle.hpp

```diff
@@ -336,7 +339,7 @@ inline typename SB_Handle::event_t SB_Handle::execute(
   /* Create the cube buffer that will hold the output of the partial gemm */
   auto cube_buffer = acquire_temp_mem < is_usm ? helper::AllocType::usm
                                                : helper::AllocType::buffer,
-       element_t > (rows * cols * depth);
+       element_out_t > (rows * cols * depth);
```

Comment:
I know this code that's being updated, but it looks like a type is being compared to a value!? What's going on?

Reply:
@s-Nick has explained this to me.